ansible/inventory

```diff
@@ -97,6 +98,7 @@ db-1.proteus.test.ooni.io
 proteus.test.ooni.io
 notify.proteus.ooni.io
 events.proteus.ooni.io
+orchestrate.ooni.io
```

Comment:
I assume `psk_orchestration` means pre-shared-key orchestration? It seems to group all orchestration hosts?

Reply:
That's what the comment says :P

Reply:
ah, it was very above in the diff. I expanded until `psk_orchestration` but no upper than that.